lib/tasks/emails.rake

```diff
@@ -207,3 +207,56 @@ task "emails:test", [:email] => [:environment] do |_, args|
       Consider changing it to 'no' before performing any further troubleshooting.
     TEXT
 end
+
+desc "run this to fix users associated to emails mirrored from a mailman mailing list"
+task "emails:fix_mailman_users" => :environment do
```

Comment:
There's even a rake task to fix wrongfully associated users 🙌 

cc @Supermathie ;)